EngineLayer/NonSpecificEnzymeSearch/NonSpecificEnzymeEngine.cs

```diff
@@ -38,7 +38,7 @@ public NonSpecificEnzymeEngine(Ms2ScanWithSpecificMass[] listOfSortedms2Scans, L
 
         protected override MetaMorpheusEngineResults RunSpecific()
         {
-            Status("In modern search engine...", nestedIds);
+            Status("In modern search engine..." + currentPartition + "/" + totalPartitions, nestedIds);
```

Comment:
Partition+1, since Partition is the index, (so it should look 1/1 instead of 0/1).
Yeah... you're right...